src/icons/index.js

```diff
@@ -180,9 +178,14 @@ import tree from "./tree.svg";
 import ubuntu from "./ubuntu.svg";
 import vault from "./vault.svg";
 import vsphere from "./vsphere.svg";
-import world from "./world.svg";
 import www from "./www.svg";
 import dots from './dots-triple.svg'
+import message from './message.svg'
```

Comment:
Can we not use react-icons for these ?

Reply:
<img width="1920" alt="Screen Shot 2022-02-07 at 11 15 16 AM" src="https://user-images.githubusercontent.com/32077311/152759586-b2358b15-7e85-4fe8-b9d1-737ffa64ac17.png">
<img width="45" alt="Screen Shot 2022-02-07 at 11 16 39 AM" src="https://user-images.githubusercontent.com/32077311/152759605-85ded12c-6a05-464e-9ed7-13c60e378f6a.png">
i don't find in react-icons same icon, they all little bit different then in design

Reply:
fixed, now message and chart icon from react-icons